toolkits/global/packages/global-skip-link/HISTORY.md

```diff
@@ -0,0 +1,4 @@
+# History
+
+## 1.0.0 (2021-05-28)
```

Comment:
I would suggest a version of `0.1.0` to start with as this isn't yet implemented on nature